### DISCUSSION TITLE
Edge-TTS for blind people

### DISCUSSION THREAD

**Discussion by BassContact** [rany2/edge-tts#84] [Ideas]
Bonjour,

 
Étant aveugle, je propose ma petite contribution à votre projet pour simplifier l’utilisation  de l’outil EDGE-TTS pour les déficients visuels.

Je suis un utilisateur Windows. J’ai installé python 3.10, et les dépendances que vous évoquez dans votre repository GitHub.

j’ai repris le travail que vous avez effectué, et j’ai créé des fichiers batch, pour chacune des voix françaises. Le tout se trouve dans un même dossier.

Le fichier batch en question reprend tout le texte contenu dans un fichier nommé : exemple.txt et génère le fichier MP3 correspondant, avec comme préfixe le nom de la voix choisie au départ.

De ce fait, il n’y a pas à se préoccupé de la syntaxe dans l’invite de commande.

Par ailleurs, l’utilisateur peut copier coller autant de texte dans le fichier exemple.txt.

Actuellement, le fichier exemple.txt contient une petite biographie du comédien français Moliere pour des besoins de test. Ce texte peut être remplacé bien entendu.

Notez qu’il est possible de faire cela pour toutes les voix, et dans toutes les langues si besoin. Désolé, je ne suis pas très bon en informatique, et je ne sais pas comment vous faire parvenir le fichier zip.

J’espère que mes explications sont claires.

 

Bien à vous, et immense merci pour ce que vous faites

**Comment by rany2** [comment]
Pensez-vous qu'une interface graphique serait utile dans votre cas par rapport aux fichiers batch ? Je n'ai jamais considéré cela comme un cas d'utilisation potentiel pour EDGE-TTS.

**Reply by deavmi** [reply]
Do you think that a graphical interface (forgot, this is some conjucation I forgot) using in your bruh.

I have never considered that as a potential use case for EDGE TTS

**Comment by BassContact** [comment]
Bonjour,

Je ne sais pas du tout. Pour ma part, je suis à l'aise avec l'invite de commande ou les batchs, mais je ne sais pas si c'est valable pour toutes les personnes aveugles.
D'où ma contribution, pour simplifier la vie de mes camarades.
Je vous ai mis le zip en pièce jointe, si vous souhaitez tester vous même.

Bien à vous,
Nordine

**Comment by rany2** [comment]
Bonjour,

Si vous le souhaitez, vous pouvez m'envoyer le fichier zip sur ranygh@riseup.net. Je ne peux pas recevoir de pièces jointes via les discussions GitHub.

**Comment by BassContact** [comment]
J'ai tenté de vous l'envoyer, mais j'ai un mail d'erreur en retour:
Sorry, we were unable to deliver your message to the following address.

***@***.***>:
550: 5.7.1 Message refused due to content violation: Email.Phishing.VOF1-6297421-0 - contact https://support.riseup.net if this is in error...

**Comment by rany2** [comment]
Bizarre, pouvez-vous essayer rany@disroot.org ?

**Comment by BassContact** [comment]
Voilà, c'est fait.
Et au cas où ca ne fonctionnerait pas, voici un lien direct :
https://we.tl/t-W0oHPWtwLR?utm_campaign=TRN_TDL_05&utm_source=sendgrid&utm_medium=email&trk=TRN_TDL_05

**Comment by rany2** [comment]
Merci !

**Comment by rany2** [comment]
To look into when I have time: [tts.zip](https://github.com/rany2/edge-tts/files/11432116/tts.zip)
